packages/discovery-provider/plugins/notifications/src/utils/populateDB.ts

```diff
@@ -347,9 +347,9 @@ export const createUSDCPurchase = async (
 ) => {
   await db
     .insert(
-      usdcPurchases.map((usdcPurchase) => ({
+      usdcPurchases.map((usdcPurchase, index) => ({
         created_at: new Date(Date.now()),
-        slot: '4',
+        slot: index,
```

Comment:
what does this do?

Reply:
`slot` is a primary key for purchases. the hardcoded slot was causing collisions when trying to add multiple purchases in the same session. this fixes :)